release_creation/bundle/requirements/v1.7.latest.requirements.txt

```diff
@@ -7,12 +7,12 @@ dbt-spark[PyHive,ODBC]~=1.7.1
 dbt-databricks~=1.7.2
 dbt-trino~=1.7.0
 dbt-fabric~=1.7.0
-dbt-synapse~=1.7.0
+dbt-synapse~=1.7.0, !=1.7.4
 dbt-rpc~=0.4.1
 google-api-core<2.16.0 # see https://github.com/dbt-labs/dbt-bigquery/issues/1081
 pandas<2.2.0,>=2.0.0
 pyasn1-modules~=0.2.1
 pyarrow~=14.0.1
 pydantic~=1.10
 pyodbc==4.0.39 --no-binary pyodbc
-snowflake-connector-python~=3.0,!=3.0.4
\ No newline at end of file
+snowflake-connector-python~=3.0,!=3.0.4
```

Comment:
I can't figure out what changed here. Is there whitespace at the end of the file?